src/_data/docs.yml

```diff
@@ -1,121 +1,237 @@
-sidebar:
+top:
   - title: Getting Started
-    default: true
-    items:
-    - title: Install
-      href: install.html
-    - title: Upgrade
-      href: upgrade.html
-    - title: "Writing a Tiltfile"
-      href: tiltfile_authoring.html
-    - title: "Tilt's Control Loop"
-      href: controlloop.html
-  - title: Tutorial
-    inSectionNav: true
+    href: /index.html
+    hrefMobile: /docs_nav_gettingstarted.html
```

Comment:
This pattern miiiight result in some slight weirdness for desktop users who have their browser at a small enough width to get the "mobile" breakpoint nav if they then resize the browser. (Alternatively, might be possible on mobile if you rotate the device.)

That said, it's enough of an edge case that it's probably not worth worrying about a ton. We can always sprinkle in some JS in the future if needed.

Reply:
Yes, I noticed this weirdness as well! And I agree that it seems like a bit of an edge case. I think some JS to finesse this layout resize behavior may be a great idea for a future PR.